packages/api/tests/api/contributions.test.ts

```diff
@@ -0,0 +1,163 @@
+import { Collection } from '@mikro-orm/core';
+import { contributions } from '../../src/api/contributions';
+import { Contribution } from '../../src/entities/Contribution';
+import { User } from '../../src/entities/User';
+import logger from '../../src/logger';
+import { testHandler } from '../testUtils/testHandler';
+
+const sampleUser: Partial<User> = {
+  name: 'Bill Nye',
+  pronouns: 'he/him',
+  schoolName: 'Science School',
+  assign: jest.fn(),
+  isAdmin: false,
+  githubId: '234234',
+  id: '1',
+  contributionList: new Collection<Contribution>(this),
+};
+
+const sampleUserNoContributionsList: Partial<User> = {
+  name: 'Bill Smith',
+  pronouns: 'he/him',
+  schoolName: 'School',
+  assign: jest.fn(),
+  isAdmin: false,
+  githubId: '0987654321',
+  id: '2',
+};
+
+const sampleSignedInUser: Express.User = {
+  profile: {
+    id: 'aaa',
+  },
+  githubToken: 'abcd123',
+};
+
+const mockQueriedUserContributions: Partial<Contribution>[] = [
```

Comment:
If you plan to use this in multiple tests, you can quickly change it to a function and then you don't have to deal with tests interfering my deleting stuff (which is honestly a better habit to get into anyways because you want mock data to be EXACTLY THE SAME without potential for modification for all tests if they share data)